experiment/flake-detector/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine:3.6
+MAINTAINER mkargaki@redhat.com
+
+RUN apk update --no-cache && \
+	apk add ca-certificates && \
+	update-ca-certificates
+
+COPY flake-detector /flake-detector
+
+ENTRYPOINT ["/flake-detector"]
```

Comment:
flakes ... :snowflake: - perhaps something about snowfall measurement or warning radar?

[Apparently accumulated snowflake depth is measured with a "snowboard"](http://www.nws.noaa.gov/os/coop/reference/Snow_Measurement_Guidelines.pdf) though I think that might have other connotations...

Canada has ["snow guages"]( https://en.wikipedia.org/wiki/Snow_gauge) but that doesn't seem much better as a name.

I think I like something like "Doppler" though, as in Doppler Radar [for measuring "flake precipitation"](https://en.wikipedia.org/wiki/Weather_radar). 
> Weather radar, also called weather surveillance radar (WSR) and Doppler weather radar, is a type of radar used to locate precipitation, calculate its motion, and estimate its type (rain, snow, hail etc.).